utils/mlir2FileCheck.py

```diff
@@ -199,7 +199,7 @@ def process_line(i, line):
 
     # Process definition of variables.
     # Special handling of function header.
-    if re.match(r'\s+(builtin\.)?func', line) is not None:
+    if re.match(r'\s+(func\.)?func', line) is not None:
```

Comment:
This is to fix generating `CHECK-LABEL`. `builtin.func` was changed to `func.func` in the current MLIR.

Reply:
tx